core/types/backwards_compat.libevm_test.go

```diff
@@ -151,6 +231,14 @@ func (e *cChainBodyExtras) RLPFieldPointersForDecoding(b *Body) *rlp.Fields {
 	}
 }
 
+func (e *cChainBodyExtras) BlockRLPFieldsForEncoding(b *BlockRLPProxy) *rlp.Fields {
+	panic("unimplemented")
+}
+
+func (e *cChainBodyExtras) BlockRLPFieldPointersForDecoding(b *BlockRLPProxy) *rlp.Fields {
+	panic("unimplemented")
```

Comment:
```suggestion
	return &rlp.Fields{
		Required: []any{&b.Header, &b.Txs, &b.Uncles, &b.Version, rlp.Nillable(&b.ExtData)},
	}
```

By the way is `rlp.Nillable(&b.ExtData)`  correct for the last field?

Reply:
I.f.f. the `coreth/core/types/Block.ExtData` field has an `rlp:"nil"` tag.